src/plugins/intel_npu/tools/protopipe/src/simulation/ov_layers_reader.cpp

```diff
@@ -151,7 +154,8 @@ InOutLayers OpenVINOLayersReader::Impl::readFromModel(const std::string& model_p
     {
         ov::preprocess::PrePostProcessor ppp(model);
 
-        const auto& input_names = extractLayerNames(model->inputs());
+        auto inputs = model->inputs();
```

Comment:
```suggestion
        ov::util::set_tensors_names(ov::AUTO, *model);
        auto inputs = model->inputs();
```
It will generate names on model inputs/outputs without names using `make_default_tensor_name` after this call using get_any_name from model inputs/outputs should be safe as all IO will have names.

Reply:
No, dev API is not available here.